scripts/pylib/twister/twisterlib/constants.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python3
+# vim: set syntax=python ts=4 :
```

Comment:
A format check has been added to compliance checks, we should not add editor specific details here.

Reply:
we also have .editconfig :)